travis-mono-test.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/sh
+
+set -e
+set -x
+
+for dir in MoreLinq.Test/bin/$CONFIGURATION/net451/*/ ; do
+	mono $dir/dotnet-test-nunit.exe $dir/MoreLinq.Test.dll
```

Comment:
No problem. However, instead of using `${dir/%}` I used `${dir}dotnet-test-nunit.exe`. I find shell variable expansion to be hard to read (I don't use shell programming everyday so I tend to forget what does each modifier does), so I prefer to avoid it if there is a more readable alternative. Moreover, I'm not sure if this is a `bash` extension or part of POSIX shell (although debian's `dash` seems to allow this particular expansion).